zoom/client.go

```diff
@@ -137,9 +144,9 @@ func (c *Client) request(ctx context.Context, method string, path string, query
 		return nil, errs.Wrap(err, "encoding URL query")
 	}
 
-	u := baseURL + path
+	url := fmt.Sprintf("%s%s", c.baseURL, path)
```

Comment:
`url` is shadowing `net/url` import from the standard library

Reply:
Yep good catch 🙏